mix.exs

```diff
@@ -56,7 +56,7 @@ defmodule Jackalope.MixProject do
 
   defp dialyzer() do
     [
-      flags: [:race_conditions, :unmatched_returns, :error_handling, :underspecs]
+      flags: [:unmatched_returns, :error_handling, :underspecs]
```

Comment:
```suggestion
      flags: [:missing_return, :extra_return, :unmatched_returns, :error_handling, :underspecs]
```

Hopefully this doesn't detect any errors on this project, but the return value checks helped on others.